platform/v1alpha1/types.go

```diff
@@ -0,0 +1,73 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// PlatformOperatorSpec defines the desired state of PlatformOperator.
+type PlatformOperatorSpec struct {
+	// PackageName contains the desired OLM-based Operator package name
+	// that is defined in an existing CatalogSource resource in the cluster.
+	//
+	// This configured package will be managed with the cluster's lifecycle. In
+	// the current implementation, it will be retrieving this name from a list of
+	// supported operators out of the redhat-operators catalog included with
```

Comment:
The redhat-operators catalog is a current implementation detail of phase 0, and we'll be allowing end users to install POs from arbitrary catalogs in future phases.

Reply:
Should we have this comment reflect how it is now or where it is going?

Reply:
I updated it to be more generic so that we should satisfy the current and future.